config/deploy.yml

```diff
@@ -0,0 +1,108 @@
+# Initially this file has some FIXMEs that you need to fill out to get started
+# with Kamal.
+#
+# This deploy.yml assumes the following prerequisites:
+#   - You have a Docker Hub account with a `docker-hub-username`.
+#   - You have a server you can SSH into.
+#   - You have a domain name that you own and will point to your server's IP address.
+#
+# Name of your application. Used to uniquely configure containers.
+service: untitled-application
+
+# Name of the container image.
+# FIXME: Change this to your Docker Hub username.
+image: docker-hub-username/untitled-application
+
+# Deploy to these servers.
+servers:
+  web:
+    - 999.999.999.999 # FIXME: The IP address of the server you can SSH into.
+  job:
+    hosts:
+      - 999.999.999.999 # FIXME: The IP address of the server you can SSH into.
+    cmd: bundle exec sidekiq
+
+# Enable SSL auto certification via Let's Encrypt and allow for multiple apps on
+# a single web server. Remove this section when using multiple web servers and
+# ensure you terminate SSL at your load balancer.
+#
+# Note: If using Cloudflare, set encryption mode in SSL/TLS setting to "Full" to
+# enable CF-to-app encryption.
+proxy:
+  ssl: true
+  host: app.yoursaas-fix-me.com # FIXME: The domain you want to use for your app. You need to create a CNAME record that points to the IP address of the server you can SSH into.
+  app_port: 3000
+
+# Credentials for your image host.
+registry:
+  # Specify the registry server, if you're not using Docker Hub
+  # server: registry.digitalocean.com / ghcr.io / ...
+  username: docker-hub-username # FIXME: The username you use to log into Docker Hub.
```

Comment:
Since `username` and `password` are paired maybe they should both go in `secrets`? Related: Will **everyone** that works on a given project **always** use the same `username`? If not, then it should definitely not be hard-coded into something checked into the repo.